gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java

```diff
@@ -97,8 +103,35 @@ public BrokerClientImpl(final GatewayCfg configuration, final ActorClock actorCl
     transport = transportBuilder.build();
     internalTransport = internalTransportBuilder.build();
 
+    final String[] parts = configuration.getCluster().getContactPoint().split(":");
+    final Integer port = Integer.parseInt(parts[1]);
+
+    atomix =
+        AtomixCluster.builder()
+            .withMemberId("gateway")
```

Comment:
The member id of the gateway has to be configurable as there could be multiple gateways in the cluster.